app/addons/documents/views.js

```diff
@@ -505,21 +517,37 @@ function(app, FauxtonAPI, Components, Documents, Databases, Views, QueryOptions)
     },
 
     beforeRender: function() {
-      var docs;
-
       this.removeNestedViews();
 
       this.pagination.setCollection(this.collection);
       this.allDocsNumber.setCollection(this.collection);
 
-      docs = this.expandDocs ? this.collection : this.collection.simple();
```

Comment:
don't we need the simple collection any more? can we delete it? for what was it for?